File: count_user_processes.py
import subprocess

# Specify the username
username = "Baha"  # Change to the user you want to check

try:
    # 'tasklist /V' lists all running processes with their username on Windows
    result = subprocess.run(["tasklist", "/V"], capture_output=True, text=True, check=True)
    processes = [line for line in result.stdout.splitlines() if username in line]

    count = len(processes)
    print(f"[OK] User '{username}' has {count} running processes.")
except subprocess.CalledProcessError:
    print("[ERROR] Failed to count processes.")

File: count_user_prosses/count_user_processes.go
package main

import (
	"fmt"
	"log"
	"os/exec"
	"strings"
)

func main() {
	username := "Baha" // Change to the user you want to check

	// Run 'tasklist /V'
	cmd := exec.Command("tasklist", "/V")
	output, err := cmd.Output()
	if err != nil {
		log.Fatal("[ERROR] Failed to get processes:", err)
	}

	lines := strings.Split(string(output), "\n")
	count := 0
	for _, line := range lines {
		if strings.Contains(line, username) {
			count++
		}
	}

	fmt.Printf("[OK] User '%s' has %d running processes.\n", username, count)
}
